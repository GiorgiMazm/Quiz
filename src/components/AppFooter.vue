<script setup lang="ts"></script>

<template>
  <div>
    <footer
      class="bg-sky-800 h-[10vh] flex items-center text-center capitalize"
    >
      <div class="container mx-auto">footer</div>
    </footer>
  </div>
</template>
