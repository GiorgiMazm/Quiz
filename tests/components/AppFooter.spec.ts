import { expect } from "vitest";
import { mount } from "@vue/test-utils";
import AppFooter from "~/components/AppFooter.vue";
describe("AppFooter", () => {
  it("Mounted correctly", () => {
    const wrapper = mount(AppFooter);
    expect(wrapper.element).toMatchSnapshot();
  });
});
