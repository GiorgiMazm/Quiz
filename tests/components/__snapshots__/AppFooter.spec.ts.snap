// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AppFooter > Mounted correctly 1`] = `
<div>
  <footer
    class="bg-sky-800 h-[10vh] flex items-center text-center capitalize"
  >
    <div
      class="container mx-auto"
    >
      footer
    </div>
  </footer>
</div>
`;
